### PR TITLE
Fresh test environment added

### PR DESCRIPTION
**Pull request by AgaBuu**

### CODE REVIEW

File: integration_tests/test.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+def adding (x, y):
+    return x + y
+
+print(adding(2, 1))
+
+
+
+def test_adding():
+    assert adding(2, 2) == 4
+
```
Comment (wrazik) [inline integration_tests/test.py:13]: You don't need empty lines at the end of the file.

File: integration_tests/test.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from cgi import test
+
+
+def adding (x, y):
+    return x + y
+
+print(adding(2, 1))
```
Comment (wrazik) [inline integration_tests/test.py:7]: You don't need print here.

File: integration_tests/test.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+from cgi import test
```
Comment (wrazik) [inline integration_tests/test.py:1]: what is cgi? Wouldn't be better to use pytest?

**Review by wrazik — CHANGES_REQUESTED** [review]
Only minor comments :) 

Remember to put "pytest" in the requirements.txt

File: integration_tests/test.py (hunk 2 above)
Comment (AgaBuu) [reply integration_tests/test.py:7]: ok

File: integration_tests/test.py (hunk 1 above)
Comment (AgaBuu) [reply integration_tests/test.py:13]: ok

File: integration_tests/test.py [hunk 4]
```diff
@@ -1,18 +1,7 @@
-from cgi import test
-
+from pytest import test
```
Comment (wrazik) [inline integration_tests/test.py:1]: You don't need this import at all! :)

**Pull request merged**